### PR TITLE
53 gifting backend

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Links/style.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const InviteLink = styled("a")(({ theme, color }) => ({
   overflowWrap: "break-word",
   "-ms-word-break": "break-all",
   wordBreak: "break-word",
+  fontSize: "14px",
```
Comment (thejoefriel) [inline client/src/components/Links/style.js:48]: This is something I need to fix across the codebase, but where we're putting in fontSizes this should be in rem. So in this case 0.875rem

File: client/src/pages/GiftTimeCredits/GiftTable.js [hunk 2]
```diff
@@ -11,12 +11,18 @@ import Tag from "../../components/Tags";
 
 import stringCutter from "../../utils/helpers/stringCutter";
 
-export const GiftTable = ({ gifts, handleCancel, preview, previewPath }) => {
+const renderCancelStatus = (rowData) => {
+  if (rowData.loading) return "caneling...";
```
Comment (thejoefriel) [inline client/src/pages/GiftTimeCredits/GiftTable.js:15]: Typo here. should be "cancelling..."

File: client/src/components/Links/style.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/components/Links/style.js:48]: done

File: client/src/pages/GiftTimeCredits/GiftTable.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/pages/GiftTimeCredits/GiftTable.js:15]: done

**Pull request merged**